Dockerfile

```diff
@@ -1,22 +1,14 @@
 FROM python:3.5-alpine
 
-RUN apk add nmap
-RUN apk add nmap-scripts
-RUN apk add git
+RUN apk add nmap nmap-scripts git
 
-RUN pip install xmltodict
-RUN pip install google-cloud-storage
-RUN pip install boto3
+RUN pip install xmltodict google-cloud-storage boto3
 
 RUN git clone https://github.com/vulnersCom/nmap-vulners /usr/share/nmap/scripts/vulners
 RUN nmap --script-updatedb
```

Comment:
I would even suggest merging these two layers since they are logically dependent:

```docker
RUN git clone https://github.com/vulnersCom/nmap-vulners /usr/share/nmap/scripts/vulners \
  && nmap --script-updatedb
```

Reply:
Thanks for this suggestion. I added this in the last commit.